## /bin/sh -e simulate dpatch for prepend_chlog.py
## h323_fix_1217.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
## DP: * Let build chan_h323 ...
## DP:   Not a simple issue because half of chan_h323 compilation units use
## DP:   OpenH323 buildsys.
## DP:   Initial patch by Tzafrir Cohen <tzafrir.cohen@xorcom.com> then 
## DP:   modified again by Guillaume Knispel <gknispel@proformatique.com> so 
## DP:   there still is a clean target and rules are made .PHONY when needed.
## DP:   Maybe related to issue: http://bugs.digium.com/view.php?id=9501

Index: asterisk-1.2.27/channels/h323/Makefile
===================================================================
--- asterisk-1.2.27.orig/channels/h323/Makefile	2008-03-19 11:32:20.000000000 +0100
+++ asterisk-1.2.27/channels/h323/Makefile	2008-03-19 11:33:47.000000000 +0100
@@ -21,8 +21,7 @@
 OPENH323DIR=$(HOME)/openh323
 endif
 
-clean:
-	rm -f *.so *.o .depend
+-include $(OPENH323DIR)/openh323u.mak
 
 $(SOURCES)::	$(SOURCES:.cxx=.cpp)
 	ln -f $< $@
@@ -31,13 +30,15 @@
 	touch $(SOURCES)
 
 libchanh323.a:	$(OBJS)
-	include $(OPENH323DIR)/openh323u.mak
 	ar crv $@ $(OBJS)
 
-Makefile.ast:	FORCE
+.PHONY: clean
+clean::
+	rm -f *.so *.o .depend
+
+.PHONY: Makefile.ast
+Makefile.ast:
 	@echo H323CFLAGS  = $(STDCCFLAGS) $(OPTCCFLAGS) $(CFLAGS) >$@.tmp
 	@echo H323LDFLAGS = $(CFLAGS) $(LDFLAGS) >>$@.tmp
 	@echo H323LDLIBS  = $(LDLIBS) $(ENDLDLIBS) $(ENDLDFLAGS) >>$@.tmp
 	@if [ -r $@ ] && cmp -s $@ $@.tmp; then rm -f $@.tmp; else mv -f $@.tmp $@; fi
-
-FORCE:
